deployment/DeploymentComponent.cpp

```diff
@@ -817,6 +822,28 @@ namespace OCL
                                 valid = false;
                             continue;
                         }
+                        if ( (*it)->getName() == "GlobalsRepository" ) {
```

Comment:
Why did you switch to `GlobalsRepository` here? Until now this `ConfigurationInterface` was only meant for constants installed by typekits. It could be used for global properties, too, but users should always be aware that write access to properties is not thread-safe.

Using the `GlobalsRepository` instead is not consistent with your changes to `TaskBrowser::printService( string name )` function, which can only print the interface of the `GlobalService`.

Reply:
Ah, I see. They are basically unrelated.
And you also added an extra `printGlobals()` method, check https://github.com/orocos-toolchain/ocl/pull/63/files#diff-5885e9d5c0b8a108d85983b9ee56334dR1555.